src/Service/Core.php

```diff
@@ -58,29 +73,15 @@ public function __construct(LoggerChannelFactoryInterface $loggerFactory,
    * @throws \Drupal\Core\Entity\EntityStorageException
    */
   public function createNodes($module) {
+    // TODO: get config from YAML.
+    // TODO: translate the node.
     $title = $this->moduleHandler->getName($module);
 
     $node = Node::create(['type' => 'applanding']);
```

Comment:
Should be using DI..

example : 
`\Drupal\demo_umami_content\InstallHelper::importArticles`
```
        // Create Node.
        $node = $this->entityTypeManager->getStorage('node')->create($values);
        $node->save();
```